src/index.ts

```diff
@@ -257,6 +258,11 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
             description: "Container MAC address (e.g., aa:bb:cc:dd:ee:ff)",
             requiresArg: false,
         })
+        .option("emulate", {
```

Comment:
maybe one last change, 

`--emulate` -> `--container-emulate` 

Thinking about having the convention of `container` prefix for all container related options

Reply:
I like the idea of a convention, i will update the code accordingly.